src/datatypes/map.c

```diff
@@ -271,7 +262,7 @@ void Map_ToString
 	ASSERT(bytesWritten != NULL);
 
 	// resize buffer if buffer length is less than 64
-	if(*bufferLen - *bytesWritten < 64) _ExtendBuffer(buf, bufferLen, 64);
+	if(*bufferLen - *bytesWritten < 64) str_ExtendBuffer(buf, bufferLen, 64);
```

Comment:
Since the caller here is `SIValue_ToString`, which passes a partially-populated `char` buffer, I think that change would be better made as part of a separate refactor.